.github/workflows/linux.yml

```diff
@@ -0,0 +1,20 @@
+name: linux
+on: [push, pull_request]
+
+jobs:
+  centos7:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        COMPILER: [gcc, llvm]
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: cvmfs-contrib/github-action-cvmfs@v2
+    - name: Start containter
+      run: |
+        docker run -it --name CI_container -v /home/runner/work/MarlinReco/MarlinReco:/Package -v /cvmfs:/cvmfs:shared -e COMPILER=${{ matrix.COMPILER }} -d ghcr.io/aidasoft/centos7:latest /bin/bash
```

Comment:
Is there a builtin variable for the package name here? It would be nice to use it because then we could re-use the same script for all iLCSoft packages

Reply:
Indeed, I just [learned](https://docs.github.com/en/actions/reference/environment-variables) you can use `GITHUB_WORKSPACE` for this.